src/react/components/pages/editorPage/editorPage.tsx

```diff
@@ -252,6 +305,11 @@ export default class EditorPage extends React.Component<IEditorPageProps, IEdito
         });
     }
 
+    private getAppPath = () => {
```

Comment:
This method is not related to the editor page and doesn't belong here.  Please move it out to a common place where it can be referenced if needed.

Reply:
Do you have a suggestion what common file ?  applicationState.ts ??